app/dashboard/notifications.py

```diff
@@ -717,12 +717,12 @@ def maybe_post_on_craigslist(bounty):
     return False
 
 
-def maybe_notify_bounty_user_removed_to_slack(bounty, username):
+def maybe_notify_bounty_user_escalated_to_slack(bounty, username, last_heard_from_user_days):
     if not settings.SLACK_TOKEN or bounty.get_natural_value() < 0.0001 or (
        bounty.network != settings.ENABLE_NOTIFICATIONS_ON_NETWORK):
         return False
 
-    msg = f"@{username} has been removed from {bounty.github_url} due to inactivity on the github thread."
+    msg = f"@vivek, {bounty.github_url} is being escalated to you, due to inactivity for {last_heard_from_user_days} days from @{username} on the github thread."
```

Comment:
We could make a `gitcoinco` moderators team of anyone that will potentially triage this type of situation (i.e. `vivek` at the moment) and `@` mention the team here versus `vivek`.

Reply:
For instance @gitcoinco/moderators

Reply:
sure.. but this is a slack message not a github message :P

Reply:
Exceptionally derp today.

Reply:
![download](https://user-images.githubusercontent.com/513929/39714588-cabaf610-51e7-11e8-9c2c-b4567ade3bba.jpg)